src/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes, Component } from 'react'
+import TimeSlice from './TimeSlice.jsx'
+import date from './utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
```

Comment:
this is impossible.  visually, when you get to more than 5 slices, they just look like gray lines